texts/icarus.txt

```diff
@@ -0,0 +1,125 @@
+‘A meteorite is reported to have landed
+
+in New England. No damage is said…’
+
+ 
+
+A littlebit of heaven fell
```

Comment:
Is the missing space a typo or poetic thing?

Reply:
It's probably a poetic thing, but some versions of the poem have a space while others don't. I'll add a space anyways since then it will make sense.

Reply:
I mean for the morphology analyser, since "littlebit" and "fatlady" aren't actual words

Reply:
Fixed it